src/index.js

```diff
@@ -199,6 +199,12 @@ module.exports = {
     'react/jsx-no-literals': 'error',
     'react/jsx-no-undef': 'error',
     'react/jsx-sort-props': 'error',
+    'react/jsx-tag-spacing': ['error', {
+      'afterOpening': 'never',
+      'beforeClosing': 'allow',
```

Comment:
This allows the following code:

```js
<div>
  {'foo'}
</div >
```

Notice the space on the closing tag.

Shouldn't this rule be set to `'never'`?